pkg/operation/care/health_check_test.go

```diff
@@ -159,7 +159,7 @@ var _ = Describe("health check", func() {
 	var (
 		ctx        = context.TODO()
 		fakeClient client.Client
-		fakeClock  *testclock.FakeClock
+		fakeClock  = testclock.NewFakeClock(time.Now())
```

Comment:
Why did you change this, we already initialize with `time.Now()` in the `BeforeEach`

Reply:
It's required. Because we can't use uninitialized variables in Table entries.